src/server/services/data-transform.service.js

```diff
@@ -4,14 +4,6 @@
  */
 
 const { logEmitter } = require("./logging.service");
```

Comment:
This looks like it's undoing the changes from release 10.0.0??

Reply:
Think these have been reversed. No changes to data-transform listed now.